strata_period_wrangler.py

```diff
@@ -66,9 +66,11 @@ def lambda_handler(event, context):
         sns_topic_arn = config['sns_topic_arn']
         sqs_message_group_id = config['sqs_message_group_id']
         sqs_queue_url = config['sqs_queue_url']
+        survey_column = event['RuntimeVariables']['survey_column']
```

Comment:
Ah, I see. It's being passed in as an env var. Will this simply be the survey code that wants to be run, then?

Reply:
This is the name of the column that the survey code lives in.